mods/ts/sequences/infantry.yaml

```diff
@@ -140,6 +140,7 @@ oxanna:
 slav:
 	Inherits: ^BasicInfantry
 	Inherits@attack: ^BasicInfantryAttack
+	icon: xxicon
```

Comment:
For reference:
>  <orabot> Mailaender, offline msg from `abcdefg30` at `2017.02.09 10:56 UTC`: https://travis-ci.org/OpenRA/OpenRA/builds/199918666#L1168 
> ...
> [13:13:12]	<Mailaender> moving the xxicon definitions onto the mod fixes things
[13:13:45]	<abcdefg30> weird stuff
[13:13:51]	<abcdefg30> yaml merging bug I suppose
[13:15:45]	<abcdefg30> hm fun, it already gives me those erros when I use "Sequences: sequences.yaml
[13:15:45]	<abcdefg30> " with an empty sequences.yaml file